server/Dockerfile

```diff
@@ -24,4 +24,4 @@ COPY --from=build /app/dist dist
 
 EXPOSE 4000
 
-CMD ["npx", "dotenv",  "-e", "./dotenv/prod.env", " -- ", "yarn", "prod"]
\ No newline at end of file
+CMD ["npx", "dotenv",  "-e", "./dotenv/.env", " -- ", "yarn", "prod"]
```

Comment:
Actually, we can feed environment variables to docker cli with --env-file option. Using `dotenv` inside docker container might confuse people.
Reference: https://docs.docker.com/engine/reference/commandline/run/

Reply:
I agree with @0916dhkim. Also, docker is not for `dev` env currently so you might want to revert this commit.

Reply:
@taehwanno Can you kindly revert these changes? I'll then quickly merge it!

Reply:
Sorry for late response. thanks!